src/main/webapp/app/shared/auth/has-any-role.directive.ts

```diff
@@ -0,0 +1,65 @@
+import { Directive, Input, OnInit, OnDestroy, TemplateRef, ViewContainerRef } from '@angular/core';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+
+import { AccountService } from 'app/core/auth/account.service';
+
+/**
+ * @whatItDoes Conditionally includes an HTML element if current user has any
+ * of the roles passed as the `expression`.
+ *
+ * @howToUse
```

Comment:
Wenn wir genau sein wollen fehlt hier die Doku für den else Fall. Aber ist okay, können wir zu einen späteren Zeitpunkt noch machen.

Reply:
erfasst in #55. merge diesen PR, damit wir den Grundstein gelegt haben.